pkg/skaffold/build/tag/tag.go

```diff
@@ -16,6 +16,9 @@ limitations under the License.
 
 package tag
 
+// ByImage maps image names to tags
+type ByImage map[string]string
```

Comment:
this name is slightly confusing to me, maybe it makes more sense as `ImageTags` or `ImageToTag`?